### ISSUE TITLE
TkDocsTest fails

### ISSUE THREAD

**Issue by caarlos0** [Nerodesk/nerodesk#182]
```
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.188 sec <<< FAILURE! - in com.nerodesk.takes.TkDocsTest
returnsListOfDocs(com.nerodesk.takes.TkDocsTest)  Time elapsed: 0.188 sec  <<< ERROR!
javax.validation.ConstraintViolationException: content can't be NULL
    at com.jcabi.aspects.aj.MethodValidator.validate(MethodValidator.java:181)
    at com.jcabi.aspects.aj.MethodValidator.beforeMethod(MethodValidator.java:95)
    at org.xembly.Directives.set(Directives.java:409)
    at com.nerodesk.takes.TkDocs.list(TkDocs.java:110)
    at com.nerodesk.takes.TkDocs.access$000(TkDocs.java:54)
    at com.nerodesk.takes.TkDocs$1.toXembly(TkDocs.java:85)
    at org.takes.rs.xe.XeChain$1.toXembly(XeChain.java:63)
    at org.takes.rs.xe.XeWrap.toXembly(XeWrap.java:57)
    at org.takes.rs.xe.XeChain$1.toXembly(XeChain.java:63)
    at org.takes.rs.xe.XeWrap.toXembly(XeWrap.java:57)
    at org.takes.rs.xe.XeAppend$1.toXembly(XeAppend.java:62)
    at org.takes.rs.xe.XeWrap.toXembly(XeWrap.java:57)
    at org.takes.rs.xe.XeChain$1.toXembly(XeChain.java:63)
    at org.takes.rs.xe.XeWrap.toXembly(XeWrap.java:57)
    at org.takes.rs.xe.RsXembly.render(RsXembly.java:109)
    at org.takes.rs.xe.RsXembly.access$000(RsXembly.java:57)
    at org.takes.rs.xe.RsXembly$1.body(RsXembly.java:94)
    at org.takes.rs.RsWrap.body(RsWrap.java:63)
    at org.takes.rs.RsWithStatus$1.body(RsWithStatus.java:88)
    at org.takes.rs.RsWrap.body(RsWrap.java:63)
    at org.takes.rs.RsWithHeader$1.body(RsWithHeader.java:109)
    at org.takes.rs.RsWrap.body(RsWrap.java:63)
    at org.takes.rs.RsWrap.body(RsWrap.java:63)
    at org.takes.facets.fork.RsFork$1.body(RsFork.java:71)
    at org.takes.rs.RsWrap.body(RsWrap.java:63)
    at org.takes.rs.RsWrap.body(RsWrap.java:63)
    at org.takes.rs.RsWrap.body(RsWrap.java:63)
    at org.takes.rs.RsPrint.printBody(RsPrint.java:158)
    at org.takes.rs.RsPrint.printBody(RsPrint.java:88)
    at com.nerodesk.takes.TkDocsTest.returnsListOfDocs(TkDocsTest.java:63)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:606)
    at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
    at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
    at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
    at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
    at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
    at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
    at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
    at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
    at org.apache.maven.surefire.junitcore.pc.Scheduler$1.run(Scheduler.java:318)
    at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
    at java.util.concurrent.FutureTask.run(FutureTask.java:262)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
    at java.lang.Thread.run(Thread.java:745)
```

My env:

```
Apache Maven 3.3.1 (cab6659f9874fa96462afef40fcf6bc033d58c1c; 2015-03-13T17:10:27-03:00)
Maven home: /usr/local/Cellar/maven/3.3.1/libexec
Java version: 1.7.0_67, vendor: Oracle Corporation
Java home: /Library/Java/JavaVirtualMachines/jdk1.7.0_67.jdk/Contents/Home/jre
Default locale: en_US, platform encoding: UTF-8
OS name: "mac os x", version: "10.10.3", arch: "x86_64", family: "mac"
```

I also tried with JDK8, same error.

**Comment by yegor256** [comment]
@caarlos0 I fixed it already in #188, let's wait for the pull request merge

**Comment by caarlos0** [comment]
@yegor256 OK, thanks!
